tt_metal/hw/ckernels/blackhole/metal/llk_api/llk_pack_api.h

```diff
@@ -213,19 +214,20 @@ inline void llk_pack_untilize(
     static_assert(diagonal == false && "Diagonal packing is not supported for BH!");
     const std::uint32_t output_id = get_output_id(output);
     std::uint32_t pack_tile_addr =
-        cb_interface[output_id].fifo_wr_ptr - 1 +
+        get_local_cb_interface(output_id).fifo_wr_ptr - 1 +
         SCALE_DATUM_SIZE(
             pack_dst_format[output_id],
             (block_c_index * ((num_faces > 2) ? num_faces / 2 : num_faces) * block_ct_dim * FACE_C_DIM)) /
             16;
 
     for (std::uint32_t block_rt = 0; block_rt < block_rt_dim; block_rt++) {
-        _llk_pack_untilize_<block_ct_dim, full_ct_dim, diagonal>(
-            pack_tile_addr, pack_dst_format[output_id], face_r_dim, num_faces, block_rt * block_ct_dim);
+        for (std::uint32_t block_rt = 0; block_rt < block_rt_dim; block_rt++) {
```

Comment:
looks likes something weird happened here

Reply:
😞 thought I fixed this already. Thanks for catching

Reply:
Fixed